guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,41 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Verify Content Checksum on container repositories using Hammer
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
+
+=== Add ACS bulk refresh option
+Added option '--all' to "hammer alternate-content-source refresh" command.
+ACS bulk refresh through API silently sanitizes input ids.
```

Comment:
```suggestion

```

I don't think this is needed, just what's on line 11 should be enough. What's on line 12 is a bug that occurred during development of 4.10.